api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/EmailMessagePayloadFactory.java

```diff
@@ -26,6 +26,28 @@
 
     private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd MMM yyyy");
 
+    private final Integer numberOfCalculationPeriods;
+
+    public static String formatPaymentAmountSummary(String summaryTemplate, int numberOfVouchers, int voucherAmountInPence) {
+        if (numberOfVouchers == 0) {
+            return "";
+        }
+        int totalAmount = numberOfVouchers * voucherAmountInPence;
+        return String.format(summaryTemplate, convertPenceToPounds(totalAmount));
+    }
+
+    public static String formatRegularPaymentAmount(int numberOfVouchers, int voucherAmountInPence) {
+        if (numberOfVouchers == 0) {
+            return "";
+        }
+        int totalAmount = numberOfVouchers * voucherAmountInPence;
```

Comment:
Can we refactor the common code between this and `formatPaymentAmountSummary` above?

Reply:
Is this better?
```

    public static String formatPaymentAmountSummary(String summaryTemplate, int numberOfVouchers, int voucherAmountInPence) {
        return formatPaymentAmount(summaryTemplate, numberOfVouchers, voucherAmountInPence);
    }

    public static String formatRegularPaymentAmount(int numberOfVouchers, int voucherAmountInPence) {
        return formatPaymentAmount("%s", numberOfVouchers, voucherAmountInPence);
    }

    private static String formatPaymentAmount(String template, int numberOfVouchers, int voucherAmountInPence) {
        if (numberOfVouchers == 0) {
            return "";
        }
        int totalAmount = numberOfVouchers * voucherAmountInPence;
        return String.format(template, convertPenceToPounds(totalAmount));
    }
```

Reply:
Yep, looks good.